stdlib/system/src/main/ballerina/src/system/system.bal

```diff
@@ -43,6 +78,24 @@ public function uuid() returns string = external;
 # + dir - The current working directory to be set to the process
 # + args - Command arguments to be passed in
 # + return - Returns a `Process` object in success, or an `Error` if a failure occurs
-public function exec(@untainted string command, @untainted map<string> env = {}, 
-                     @untainted string? dir = (), @untainted string... args) 
-                     returns Process|Error = external;
+public function exec(@untainted string command, @untainted map<string> env = {},
+                     @untainted string? dir = (), @untainted string... args) returns Process|Error {
+    return nativeExec(java:fromString(command), env, dir, args);
+}
+
+function nativeExec(handle command, map<string> env, string? dir, string[] args) returns Process|Error = @java:Method {
+    name: "exec",
+    class: "org.ballerinalang.stdlib.system.nativeimpl.Exec"
+} external;
+
+function covertToString(handle value) returns string {
+    string output = "";
+    var result = java:toString(value);
+    if (result is string) {
+        output = result;
```

Comment:
We can remove the variable `output` and return the result directly right ?

Reply:
Fixed

Reply:
This isn't fixed right ?

Reply:
Still the `output` variable is there, right?

Reply:
This was not fixed.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/20113/